wp-content/themes/currentorg/js/current-ltw-projects.js

```diff
@@ -1 +1,54 @@
+var $ = jQuery;
+$(document).ready(function(){
 
+    current_ltw_projects_load_param_project_if_exists();
+    
+    $( ".projects-list article .entry-title a" ).click(function(e){
+        e.preventDefault();
+        var post_id = $(this).data("post-id");
+        current_ltw_projects_load_single_project(post_id);
+    });
+
+});
+
+/**
+ * Function to load the single project partial for whatever project
+ * is provided.
+ * 
+ * @param int post_id The id of the project we want to load
+ */
+function current_ltw_projects_load_single_project( post_id ) {
+
+    // give a reassuring "loading" notice
+    $(".projects-single-holder").html("Loading project...");
+
+    // actually load the single project
+    $(".projects-single-holder").load(ajax_object.ajax_url+"?action=load_more_post&post_id="+post_id);
```

Comment:
Consider https://current.test/ltw-test-page/ and the per-project page https://current.test/ltw-test-page/?project_id=3251741:

On page load, https://current.test/ltw-test-page/ displays in the single-project area the final post from the queue, which is the last item in the post list.

The DOM looks like this:

```html
<div class="projects-single-holder">
	<h1 class="projects-single-title">Youth Reporting Institute</h1>
	....
</div>
```
Styles on `.projects-single-holder` give the project a single black outline.

Clicking on the last item in the list loads the post:

```html
<div class="projects-single-holder">
	<div class="projects-single-holder">
		<h1 class="projects-single-title">Youth Reporting Institute</h1>
		....
	</div>
</div>
```

The post now has a double black outline.

Loading the project-specific URL https://current.test/ltw-test-page/?project_id=3251741 means that that project is, upon page load, queried for by JS and then loaded by JS:

```html
<div class="projects-single-holder">
	<div class="projects-single-holder">
		<h1 class="projects-single-title">Youth Reporting Institute</h1>
		....
	</div>
</div>
```

The post now has a double black outline.

In conclusion: loading the project via JS means that we get an extra `div.projects-single-holder`. 

- [ ] should we have separate divs for inner and outer, so that the AJAX response can reply with only the inner div to be appended to the outer? This might require splitting `partials/project-single-holder.php` into two files: the holder and the held.

Reply:
@benlk I noticed that too and wanted to hear your opinion on what you think would be best. I think splitting up the "holder" and the "held" sounds like a viable solution.

Reply:
https://github.com/INN/umbrella-currentorg/pull/140#issuecomment-637046748 explores a little more the holder/held distinction; I think what we'd end up with is:

```
project-single-layout
    project-single-close
    project-single-holder
         article.project-single-held
```

Reply:
@benlk https://github.com/INN/umbrella-currentorg/pull/140/commits/8a616c2990e8cfef9a1a95f9b3be0289d840ea7d starts the implementation of that structure